src/components/ButtonAddnewBook.jsx

```diff
@@ -0,0 +1,27 @@
+import { useDispatch } from 'react-redux';
+import { AddBook } from '../redux/books/bookSlice';
+
+const ButtonAddnewBooks = () => {
+  const dispatch = useDispatch();
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const title = event.target.title.value;
+    const author = event.target.author.value;
+    const newbook = { title, author };
+    dispatch(AddBook(newbook));
+    event.target.reset();
```

Comment:
- [ ] [REQUIRED] Kindly use the useState hook for `title` and `author`, which is updated through the `onChange` event as stated by the project requirement. Such that the initial and final state looks like this :
```jsx
 const [title, setTitle] = useState("")
 const [author, setAuthor] = useState("")
```

Reply:
Thank you for taking the time to explain me!

Reply:
You are welcome @Nkaleth 👍🏽 
Happy coding 🥇